app/views/video_chats/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="video-chat-wrapper" id="chat" data-token="<%= @token %>" data-channel="<%= params[:id] %>">
+
+</div>
+<h1>This is an experiment</h1>
+<h3>It's only a partially implemented view.</h3>
+<h4>But this could be a more straightforward way to do video chat within /connect.</h4>
+<%# TODO Make paramsp[:id] map to a chat channel and ensure the users have permission %>
```

Comment:
Before we do this do we want to restrict what user's can use this feature? Like maybe only trusted users or something.